packages/app-desktop/InteropServiceHelper.ts

```diff
@@ -78,6 +78,7 @@ export default class InteropServiceHelper {
 					shim.setTimeout(async () => {
 						if (target === 'pdf') {
 							try {
+								win.webContents.executeJavaScript('document.querySelectorAll(\'details\').forEach(el=>el.setAttribute(\'open\',\'\'))');
```

Comment:
Please add a comment above this line to explain what it does, and link to the GitHub issue